src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -408,8 +408,8 @@
     <key alias="newPartialViewFromSnippet">Ny partial view fra snippet</key>
     <key alias="newPartialViewMacroFromSnippet">Ny partial view makro fra snippet</key>
     <key alias="newPartialViewMacroNoMacro">Ny partial view makro (uden makro)</key>
-    <key alias="newStyleSheetFile">Ny stylesheet-fil</key>
-    <key alias="newRteStyleSheetFile">Ny Rich Text Editor stylesheet-fil</key>
+    <key alias="newStyleSheetFile">Ny stylesheet fil</key>
+    <key alias="newRteStyleSheetFile">Ny Rich Text Editor stylesheet fil</key>
```

Comment:
I'd argue that `stylesheet-fil` was correct (hence my earlier comment about compound words).

Reply:
I'm torn on this one 😬 - how about _"Nyt stylesheet"_ (and _"Nyt Rich Text Editor stylesheet"_)? Sounds much more like the way we'd actually refer to it (as a stylesheet "thing", instead of a file of type stylesheet).